sdk/ml/azure-ai-ml/azure/ai/ml/constants/_common.py

```diff
@@ -311,6 +311,7 @@ class ArmConstants:
     BASE_TYPE = "base"
     WORKSPACE_BASE = "workspace_base"
     WORKSPACE_PARAM = "workspace_param"
+    ROLE_ASSIGNMENTS = "roleAssignments"
```

Comment:
should this align with the naming convention of other contsant values? role_assignments